### DISCUSSION TITLE
Episode 194: The One Thing Every Programmer Should Know with Kevlin Henney

### DISCUSSION THREAD

**Discussion by codereport** [codereport/adsp2#93] [Episode]
Episode Link: https://adspthepodcast.com/2024/08/09/Episode-194.html

Feel free to write comments or ask questions 🙂

**Comment by hypodronic** [comment]
Perfect Bryce moment: "Well we'd better wrap up as we've gone long and [eight minute digression, mercifully 10xed]"
